acceptance/tests/validate_vendored_ruby.rb

```diff
@@ -9,7 +9,8 @@ def package_installer(agent)
   when /aix/
     lambda { |package| on(agent, "rpm -Uvh #{package}") }
   when /solaris-10/
-    lambda { |package| on(agent, "opt/csw/bin/pkgutil -y -i #{package}") }
+    echo "mirror=http://www.gtlib.gatech.edu/pub/OpenCSW/stable" > /var/tmp/vanagon-pkgutil.conf
```

Comment:
I think you'll probably want to update the pkgutil.conf somewhere here:
https://github.com/puppetlabs/puppet-agent/blob/master/acceptance/tests/validate_vendored_ruby.rb#L59

As it is, that will fail the test b/c Ruby will think of `echo` as a function, i.e. this part can be left alone.